src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -29,11 +29,16 @@
 final class MethodArgumentSpaceFixer extends AbstractFixer implements ConfigurableFixerInterface
 {
     /**
+     * @var array<string, bool>
+     */
+    private static $defaultConfiguration = array('keepMultipleSpacesAfterComma' => false);
```

Comment:
introduction of this is in line with the work done in this PR https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2382